styles/styles.css

```
@@ -0,0 +1,270 @@
+@font-face{
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face{
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    src: url("../fonts/ptsans-700.woff2") format("woff2");
+    font-display: swap;
+}
+
+body{
+    font-family: "PT Sans", sans-serif;
+    background-color: #ffffff;
+    color: #000000;
+    font-size: 18px;
+    line-height: 21px;
+    font-weight: 400;
+}
+
+/* header */
+.page-header{
+    font-size: 20px;
+    line-height: 26px;
+    background-color: #ffffff;
+    font-weight: 700;
+}
+
+.navigation-item{
+    list-style: none;
+}
+
+.navigation-link{
+    color: #000000;   
+    text-decoration: none;
+}
+/* Стиль для кнопок */
+.button{
+    background-color: #756257;
+    color: white;
+    font-size: 16px;
+    line-height: 20px;
+    text-transform: uppercase;
+}
+
+.intro{
+    background: url(../images/background/decoration.svg) no-repeat center bottom/100% auto, 
+    url(../images/background/intro-background.jpg) no-repeat center/100% auto;
+}
```

Декорацию лучше оставить на потом и сделать CSS

Текст задания:
> У каждого фона должен быть контрастный цвет текста.

Это нужно на случай, если картинка не загрузится. У тебя этого нет.
Т.е. нужно добавить цвет. 
```suggestion
.intro{
    background: url(../images/background/intro-background.jpg) no-repeat center/100% auto #182b32;
}
```

Я бы советовал использовать не сокращённые свойства. Так проще поддерживать